src/components/SellerList.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const SellerList = props => {
+  return (
+    <AllWrap>
+      <SellerTotalBox>
+        <SellerTotal>681명</SellerTotal>
+      </SellerTotalBox>
+      <SellerBoxWrap>
+        <SellerBox
+          onClick={() => {
+            props.history.push('/seller');
+          }}
+        >
+          <SellerImageBox>
+            <SellerImg src="images/sellerImg.jpg" alt="" />
+          </SellerImageBox>
+
+          <SellerNameBox>
+            <SellerName>최홍자</SellerName>
+          </SellerNameBox>
+        </SellerBox>
+        <SellerBox>
+          <SellerImageBox>
+            <SellerImg src="images/sellerImg.jpg" alt="" />
+          </SellerImageBox>
+
+          <SellerNameBox>
+            <SellerName>최홍자</SellerName>
+          </SellerNameBox>
+        </SellerBox>
+        <SellerBox>
+          <SellerImageBox>
+            <SellerImg src="images/sellerImg.jpg" alt="" />
+          </SellerImageBox>
+
+          <SellerNameBox>
+            <SellerName>최홍자</SellerName>
+          </SellerNameBox>
+        </SellerBox>
+
+        <SellerBox>
+          <SellerImageBox>
+            <SellerImg src="images/sellerImg.jpg" alt="" />
+          </SellerImageBox>
+
+          <SellerNameBox>
+            <SellerName>최홍자</SellerName>
+          </SellerNameBox>
+        </SellerBox>
+
+        <SellerBox>
+          <SellerImageBox>
+            <SellerImg src="images/sellerImg.jpg" alt="" />
+          </SellerImageBox>
+
+          <SellerNameBox>
+            <SellerName>최홍자</SellerName>
+          </SellerNameBox>
+        </SellerBox>
+        <SellerBox>
+          <SellerImageBox>
+            <SellerImg src="images/sellerImg.jpg" alt="" />
+          </SellerImageBox>
+
+          <SellerNameBox>
+            <SellerName>최홍자</SellerName>
+          </SellerNameBox>
+        </SellerBox>
+      </SellerBoxWrap>
+    </AllWrap>
+  );
+};
+
+const AllWrap = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+`;
+
+const SellerTotalBox = styled.div`
+  width: 1190px;
+  height: 40px;
+  display: flex;
+  align-items: center;
+`;
+
+const SellerTotal = styled.h2``;
+
+const SellerBoxWrap = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 1190px;
+`;
+
+const SellerBox = styled.div`
+  margin-bottom: 30px;
+`;
+
+const SellerImageBox = styled.div`
+  width: 190px;
+  height: 190px;
+`;
+
+const SellerImg = styled.img`
+  width: 100%;
+  height: 100%;
```

Comment:
- 이미지 자체에 바로 크기를 부여하셔도 될 것 같습니다!
```suggestion
  width: 190px;
  height: 190px;
```